lib/request.js

```diff
@@ -17,7 +17,7 @@ const only = require('only');
  * Prototype.
  */
```

Comment:
The comment `Prototype` should go since this is not a prototype object anymore, but a class (function) that _has_ a prototype.